### ISSUE TITLE
Aiming Upwards Scope Glitch

### ISSUE THREAD

**Issue by TeamIlluminati** [chriskvm/ShellshockIoAimbot#18]
When you aim at an enemy up higher than you (say 3ish levels) with the freeranger, it aims below the egg, making the aimbot function useless as it aims below the egg. Perhaps make it prefer the middle of the egg if it's not covered, and if it's covered aim toward the part of the egg that isn't covered. Not sure if that makes any sense or how that could be easily implemented though.

**Comment by chriskvm** [comment]
This sounds like #12. Was he on an incline?  It's because I'm trusting what the game says the velocity is. I should be storing the velocity myself.

**Comment by TeamIlluminati** [comment]
Yeah. However, storing it yourself could cause a performance loss to the client.
